src/main/java/org/folio/helper/PurchaseOrderLineHelper.java

```diff
@@ -896,6 +898,36 @@ private CompletableFuture<Void> processPoLineEncumbrances(CompositePurchaseOrder
     return completedFuture(null);
   }
 
+  private CompletableFuture<Void> updateInventoryItemStatus(CompositePoLine compOrderLine, RequestContext requestContext) {
+    return purchaseOrderLineService.getPoLinesByOrderId(compOrderLine.getPurchaseOrderId(), requestContext)
+      .thenCompose(poLines -> {
+        if (poLines.size() > 1) {
+          List<String> poLinesId = poLines.stream().map(PoLine::getId).collect(toList());
+          return inventoryManager.getItemsByPoLineIdsAndStatus(poLinesId, ItemStatus.ON_ORDER.value(), requestContext)
+            .thenCompose(items -> {
+              if (items.size() > 1) {
+                Optional<JsonObject> poLineItem = items.stream()
```

Comment:
Do we really need to update only one item with ON ORDER status. What about other items which have the same status?

Reply:
There was a call with Dmitry. The order always has one item with the status ON ORDER.

Reply:
Thanks for info